build-system/tasks/build.js

```diff
@@ -51,6 +52,7 @@ async function build() {
   maybeUpdatePackages();
   const handlerProcess = createCtrlcHandler('build');
   await performBuild();
+  await formatExtractedMessages();
```

Comment:
Move this from `build()` to `performBuild()` (so it's picked up by `gulp watch` as well).